invenio_opendata/base/templates/termsofuse.html

```diff
@@ -52,16 +52,17 @@ <h4 class="">Terms of use</h4>
 			<div class="col-md-12">
 				<div class="header">CERN Open Data Terms of Use</div>
 				<div class="body">
+					<h4>The CERN Open Data Portal is offered by CERN as part of its mission to publish the results of its experimental and theoretical work.</h4>
 					<h4>Use of the CERN Open Data Portal service denotes agreement with the following terms:</h4>
 					<ol>
 						<li>The CERN Open Data Portal is provided free of charge.</li>
 						<li>All metadata and datasets in the CERN Open Data Portal are made available under the terms of the CC0 waiver.</li>
-						<li>All other content is made available under its applicable license conditions as specified on the relevant web pages. For example, software packages are usually licensed under GNU General Public License (GPL). Download and use of such other content from the CERN Open Data Portal does not, unless expressly stated in the applicable license conditions, transfer any intellectual property.</li>
-						<li>Content of the CERN Open Data Portal will be updated or otherwise modified on a regular basis. The CERN Open Data Portal reserves the right to alter or delete content without notice.</li>
+						<li>All other content is made available under its applicable license conditions as specified on the relevant web pages. For example, software packages are usually licensed under GNU General Public License (GPL). Download and use of such other content from the CERN Open Data Portal does not, unless expressly stated in the applicable license conditions, transfer any intellectual property. Bulk downloading of email addresses taken from CERN Open Data Portal is not allowed.</li>
```

Comment:
|  Bulk downloading of email addresses [...]

There are no email addresses to download from the COD portal...